server/externs/multer.js

```diff
@@ -13,3 +13,17 @@ function multer(params) {}
  * @param {string} arg
  */
 multer.prototype.single = function(arg) {};
+
+/**
+ * @typedef {{
+ *   filedname: string,
+ *   originalname: string,
+ *   encoding: string,
+ *   mimetype: string,
+ *   destination: string,
+ *   path: string,
+ *   filename: string,
+ *   size: number
+ * }}
+ */
+multer.prototype.file;
```

Comment:
multer.File. Uppercase the first letter indicates the type is a class-like or struct-like object. Lowercasing means the defined stuff is more like a namespace.

Please also add EOL.